iptables/iptables.go

```diff
@@ -330,12 +357,19 @@ func Exists(table Table, chain string, rule ...string) bool {
 	if supportsCOpt {
 		// if exit status is 0 then return true, the rule exists
 		_, err := Raw(append([]string{"-t", string(table), "-C", chain}, rule...)...)
+		if err != nil {
+			_, err = Raw(append([]string{"-t", string(table), "-C", chain}, append(rule, commentArgs...)...)...)
+		}
 		return err == nil
 	}
 
 	// parse "iptables -S" for the rule (it checks rules in a specific chain
 	// in a specific table and it is very unreliable)
-	return existsRaw(table, chain, rule...)
+	if existsRaw(table, chain, rule...) {
```

Comment:
But I don't test it via old iptable version which don't support `-C` check. I currently don't have a environment. Can you confirm it ?